ComfyUI/Dockerfile.local-cpu

```
@@ -17,71 +26,83 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
+    echo "Workflow ${COMFYUI_PATH}/user/default/workflows/\$1.json not found"
+    ls -la ${COMFYUI_PATH}/user/default/workflows/
+fi
+comfy node fix all
+find . -mindepth 2 -type d -name ".git" -exec rm -rf {} +
+find . -mindepth 2 -type f -name ".gitignore" -exec rm -f {} +
+git add --all
+git commit -m 'All changes'
+git diff --diff-filter=d --submodule=diff --name-only HEAD^ \
+  | tar -czf changeset/\${1:-snapshot}.tar.gz -T -
+EOF
+RUN chmod +x save-changeset.sh
```

Hadolint (2.12.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)